src/content/1.7/development/page-reference/back-office/product/form/_index.md

```diff
@@ -28,50 +28,203 @@ This form combines different sub form types, each one is handled by dedicated CQ
 
 | Form type                | Field      | Description                                              | CQRS Command                                           |
 |:-------------------------|:-----------|:---------------------------------------------------------|:-------------------------------------------------------|
-| **BasicInformationType** | `basic` | Contains the basic product information of the Product | `AddProductCommand` and `UpdateBasicInformationCommand` |
-| **DescriptionType** | `description`    | Contains the description of the Product | `UpdateDescriptionCommand` |
-| **ShortcutType** | `shortcut`    | Contains shortcut for prices and quantity of the Product | `UpdatePricesCommand`, `UpdateStockCommand` and `UpdateOptionsCommand` |
-| **RelationshipsType**    | `relationships` | Contains relationships of the product | N/A |
-| **ShortcutType**         | `shortcuts`     | Contains shortcut fields editable in other sub forms but accessible in first tab easily | N/A |
-| **TypeaheadProductPackCollectionType** | `pack_items` | List of products (for Pack of product) | `AddProductToPackCommand` `UpdateProductPackCommand` `RemoveProductFromPachCommand` |
+| **BasicInformationType** | `basic` | Contains the basic product information of the Product | `AddProductCommand` and `UpdateProductBasicInformationCommand` |
+| **DescriptionType** | `description`    | Contains the description of the Product | `UpdateProductBasicInformationCommand` |
+| **ShortcutType** | `shortcut`    | Contains shortcut for prices, quantity and reference of the Product | `UpdateProductPriceCommand`, `UpdateProductStockCommand` and `UpdateProductOptionsCommand` |
+| **TypeaheadProductPackCollectionType** | `pack_items` | List of products (for Pack of product) | `UpdateProductPackCommand` `CleanProductFromPackCommand` |
+| **RelationshipsType** | `relationships` | Contains relationships of the product | (see [RelationshipsType](#relationshipstype)) |
+| **PriceType** | `price` | Contains data about pricing | `UpdateProductPriceCommand` |
+| **SpecificPriceType** | `add_specific_price` | Component to add a specific price | `AddProductSpecificPriceCommand` |
+| **SpecificPricePriorityType** | `specific_price_priority` | Contains data about the priority order for specific prices | `UpdateSpecificPricePriorityCommand` |
+| **StockType** | `stock` | Contains data about the product stock | `UpdateProductStockCommand` `UpdateProductWarehouseLocationCommand` (TBD should it be managed in stock form or separately) |
```

Comment:
`UpdateProductWarehouseLocationCommand` there is still some logic related to warehouses ?

Reply:
It seems to be still managed, I guess it is for backward compatibility because some Modules use it Just like advanced shop management It would be nice to have some sample modules to test this feature